packages/graphql-tool-utilities/src/tests/schema.json

```diff
@@ -0,0 +1,15581 @@
+{
```

Comment:
The prettier doesn't seem to like JSON files, what can I do to resolve the CI errors?